index.html

```diff
@@ -92,7 +92,7 @@
 				<h1 class="jumbotron-heading">Open Source Diversity</h1>
 				<p class="lead">For more diversity & inclusion in free & open source software communities 😊</p>
 				<p>
-					<a href="https://discourse.opensourcediversity.org/" class="btn btn-primary" target="_blank" rel="noopener"><img src="img/discourse.svg" /> Join our forum!</a>
+					<a href="https://discourse.opensourcediversity.org/" class="btn btn-primary" target="_blank" rel="noopener"><img alt="" src="img/discourse.svg" /> Join our forum!</a>
```

Comment:
thank you @2201monesh for your contribution especially making the site more ally compliant, perhaps we can also add an alt text here rather than an empty string :)

Reply:
@2201monesh are you still interested in working on this?